api/routes/recipes.route.js

```diff
@@ -6,7 +6,7 @@ const RecipesController = require('../controllers/recipes.controller');
 
 router.get('/', RecipesController.getAllRecipes);
 
-router.post('/', RecipesController.createRecipe);
+router.post('/create', RecipesController.createRecipe);
```

Comment:
A `post` verb usually means you're creating something. The `/create` is redundant.